test/ui/OmniboxTest.ts

```diff
@@ -166,6 +167,60 @@ export function OmniboxTest() {
         expect(test.env.searchEndpoint.listFieldValues).toHaveBeenCalled();
       });
 
+      it('enableFieldAddon should always provide suggestions with field leading with @', async done => {
+        test = Mock.optionsComponentSetup<Omnibox, IOmniboxOptions>(Omnibox, {
+          enableFieldAddon: true
+        });
+
+        test.cmp.setText('@');
+        const fieldsDescription: IFieldDescription[] = [
+          {
```

Comment:
I think a factory function that returns a fieldDescription object would make these two tests visually shorter.

```
function buildFieldDescription(name:string) {
    return {name, includeInQuery: true, groupByField: true} as IFieldDescription;
}
```